invenio_cli/cli.py

```diff
@@ -20,19 +20,22 @@
 
 
 @click.group()
+@click.version_option()
 def cli():
     """Initialize CLI context."""
 
 
 @cli.command()
 @click.option('--flavour', type=click.Choice(['RDM'], case_sensitive=False),
               default='RDM', required=False)
-def init(flavour):
+@click.option('--cookiecutter', nargs=2, required=False,
+              help='Pass <cookiecutter git url> <branch> to select cookiecutter template')  # noqa
```

Comment:
Is the url really needed? Is this intended to test cookiecutter fork urls?

Reply:
For forks is a good idea. I see a potential problem on people testing random stuff, but this is meant for us and many times we have stuff on our fork...

A quick minor:

```suggestion
              help='Pass <cookiecutter git url> <branch/tag> to select cookiecutter template')  # noqa
```

Reply:
Ah I see! Making it work for local cookiecutters would be even more useful. I will do that.